io.github.Adda0.Stardrop.metainfo.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>io.github.adda0.stardrop</id>
```

Comment:
Is it `adda0` or `Adda0`?

Reply:
It's case sensitive here and has to match the filename.

```suggestion
  <id>io.github.Adda0.Stardrop</id>
```

Reply:
The suggested fix was applied.